src/bindings_v5_0_0/virtio_scsi.rs

```diff
@@ -0,0 +1,803 @@
+/* automatically generated by rust-bindgen */
```

Comment:
Could you please help to document the bindgen command to generate this file?

Reply:
I used the same command as documented in [CONTRIBUTING](https://github.com/rust-vmm/virtio-bindings/blob/master/CONTRIBUTING.md) that is `bindgen /usr/include/linux/virtio_scsi.h -o virtio_scsi.rs --with-derive-default --with-derive-partialeq -- -Iinclude`. Should I include that in the comment?

Reply:
thanks, please add a comment for it:)